client/my-sites/plans-features-main/test/plan-type-selector.jsx

```diff
@@ -17,25 +24,36 @@ describe( '<PlanTypeSelector />', () => {
 	};
 
 	test( 'Should show CustomerTypeToggle when kind is set to `customer`', () => {
-		const comp = shallow(
-			<PlanTypeSelector { ...myProps } kind="customer" customerType="personal" />
-		);
+		render( <PlanTypeSelector { ...myProps } kind="customer" customerType="personal" /> );
+
+		expect( screen.getByRole( 'radiogroup' ) ).toHaveClass( 'is-customer-type-toggle' );
+
+		const radios = screen.queryAllByRole( 'radio' );
 
-		expect( comp.find( 'CustomerTypeToggle' ).length ).toBe( 1 );
+		expect( radios[ 0 ] ).toHaveTextContent( 'Blogs and personal sites' );
+		expect( radios[ 1 ] ).toHaveTextContent( 'Business sites and online stores' );
 
-		// customerType prop is passed to the child component
-		expect( comp.find( 'CustomerTypeToggle[customerType="personal"]' ).length ).toBe( 1 );
+		expect( screen.queryByRole( 'radio', { checked: true } ) ).toBe( radios[ 0 ] );
```

Comment:
Or we already have `radios[0]` and can directly assert `toBeChecked()` on it, avoiding an extra query.

Reply:
Good catch 👍🏻